tox.ini

```diff
@@ -50,4 +50,5 @@ passenv=
 commands =
     # for doc generate we need to install this moudle that can `import greenplumpython`
     pip install --force-reinstall .
+    pip install --force-reinstall psycopg2-binary
```

Comment:
can you add some comments why we need psycopg2-binary here

Reply:
Yes, I will explain it fix problem for macos when building docs in local evironment